### PR TITLE
bump deps

### PR DESCRIPTION
**Pull request by stevemao**
No breaking changes to this modules.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,16 +47,16 @@
   "dependencies": {
     "ansi-styles": "^2.1.0",
     "escape-string-regexp": "^1.0.2",
-    "supports-color": "^2.0.0"
+    "supports-color": "^3.1.2"
```
Comment (Qix-) [inline package.json]: @sindresorhus didn't we want to hold off on bumping this for a while?

Comment (sindresorhus) [inline package.json]: Yes, but it's ok. The next chalk version will be a major anyways.

**Comment by Qix-** [comment]
Where are the tests for this? Did github mess up?

screen shot 2016-01-07 at 15 26 46

**Comment by jbnicolai** [comment]
Apparently it's hidden under the 'view details' toggle after a merge (didn't know either).


[cid:21fe9423-673f-4cd6-abe0-2ec5eb2093a2]


________________________________

**Comment by Qix-** [comment]
@jbnicolai derp, thanks. Must be a new thing.

**Pull request merged**